src/tasks/Microsoft.NET.Sdk.WebAssembly.Pack.Tasks/GenerateWasmBootJson.cs

```diff
@@ -97,6 +100,7 @@ public void WriteBootJson(Stream output, string entryAssemblyName)
         {
             resources = new ResourcesData(),
             startupMemoryCache = ParseOptionalBool(StartupMemoryCache),
+            aot = WasmAotFlag,
```

Comment:
Please, move the setting to the `if(IsTargeting80OrLater())` section below and set it only if true, so the boot config doesn't contain default values